README.md

```diff
@@ -1,70 +1,186 @@
+To get a local copy up and running, follow these steps:
+- Create a local directory whre you can clone the project
+- Clone the project to your directory by running
+  - `git clone https://github.com/iamchristianani/cryptobillia.git`
+- Install all dependencies by running
+  - `npm install`
+- Open project in browser by running
+  - `npm start`
+  
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
 
-### Advanced Configuration
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In order to run this project you need:
+
+- Git and GitHub
+- A code editor
+- NodeJs
+- A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Create a local directory whre you can clone the project
+- Clone the project to your directory by running
+  - `git clone https://github.com/iamchristianani/cryptobillia.git`
+- Install all dependencies by running
+  - `npm install`
+- Open project in browser by running
+  - `npm start`
+
+### Install
+
+Install this project with:
+
+- You can use npm or yarn
+  - `npm install`
+
+
+### Run tests
+
+To run tests, run the following command:
+
+- `npm run test`
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+- GitHub Pages
+- Netlify
+- Render
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Christian Ani**
+
+- GitHub: [@iamchristianani](https://github.com/iamchristianani)
+- Twitter: [@kriznode](https://twitter.com/kriznode)
+- LinkedIn: [Christian Ani](https://www.linkedin.com/in/anikriz/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improve UI**
+- [ ] **Create a visual representations of the data**
+- [ ] **Add More Functionalities**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/iamchristianani/cryptobillia/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly leave a ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the great resources shared.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Kindly add an acknowledgement to Nelson Sakwa whose design inspired the project UI design